sys/dev/ahci/ahci_pci.c

```diff
@@ -684,3 +684,5 @@ static driver_t ahci_ata_driver = {
         sizeof(struct ahci_controller)
 };
 DRIVER_MODULE(ahci, atapci, ahci_ata_driver, ahci_devclass, NULL, NULL);
+MODULE_PNP_INFO("U32:device;U8:rev;D:#", pci, ahci, ahci_ids,
```

Comment:
As to your email question, I don't see an issue with using `rev` in the description string, but I'll defer to @bsdimp

Reply:
Moved it.